bin/postgres-ha/bootstrap-postgres-ha.sh

```diff
@@ -120,11 +121,33 @@ primary_initialization_monitor() {
 remove_patroni_pause_key()  {
     if [[ -f "${PATRONI_POSTGRESQL_DATA_DIR}/patroni.dynamic.json" ]]
     then
-        echo "Now removing \"pause\" key from patroni.dynamic.json configuration file if present"
+        echo_info "Now removing \"pause\" key from patroni.dynamic.json configuration file if present"
         sed -i -e "s/\"pause\":\s*true,*\s*//" "${PATRONI_POSTGRESQL_DATA_DIR}/patroni.dynamic.json"
     fi
 }
 
+# Checks to see if a PostgreSQL server (v12 or above) is configured for a PITR recovery. This is
+# done by checking whether or not a 'recovery.signal' file is present, along with whether or not
+# 'recovery_target' settings are present in the 'postgresql.auto.conf' file (as configured by
+# pgBackRest during a restore).
+is_pg_in_pitr_recovery() {
+    [[ -f "${PATRONI_POSTGRESQL_DATA_DIR}/recovery.signal" ]] && has_recovery_target "postgresql.auto.conf"
+}
```

Comment:
Far be it from me to be anywhere close to a decent shell scripter, but are we able to combine this into something like:

```shell
([[ -f "${PATRONI_POSTGRESQL_DATA_DIR}/recovery.signal" ]] && has_recovery_target "postgresql.auto.conf") ||
([[ -f "${PATRONI_POSTGRESQL_DATA_DIR}/recovery.conf" ]] && has_recovery_target "recovery.conf")
```

Reply:
These can def combined as suggested.  The primary motivation for the current design, however, was really just to cleanly separate pre-PG 12 PITR detection logic from that for PG 12 and greater (e.g. in case they ever need to be independently changed or end up evolving separately).

Reply:
I don't mind the separation. The difference in v12 is significant and worth distinction, comments, etc. Recovery settings can come from lots of files or be interrogated using `postgres -C`.

Reply:
Left as separate funcs for the time being.  Just let me know if any concerns.